nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/main/java/org/apache/nifi/hbase/PutHBaseCell.java

```diff
@@ -96,16 +97,18 @@ protected PutFlowFile createPut(final ProcessSession session, final ProcessConte
 
 
         final byte[] buffer = new byte[(int) flowFile.getSize()];
-        session.read(flowFile, new InputStreamCallback() {
-            @Override
-            public void process(final InputStream in) throws IOException {
-                StreamUtils.fillBuffer(in, buffer);
-            }
-        });
+        session.read(flowFile, in -> StreamUtils.fillBuffer(in, buffer));
 
+        PutColumn column = null;
+        if (visibilityStringToUse != null && !visibilityStringToUse.equals("")) {
```

Comment:
Cosmetic comment - feel free to ignore.
````java
PutColumn column = StringUtils.isEmpty(visibilityStringToUse)
    ? new PutColumn(columnFamily.getBytes(StandardCharsets.UTF_8),
                    columnQualifier.getBytes(StandardCharsets.UTF_8), buffer, timestamp)
    : new PutColumn(columnFamily.getBytes(StandardCharsets.UTF_8),
                    columnQualifier.getBytes(StandardCharsets.UTF_8), buffer, timestamp, visibilityStringToUse);
````

Reply:
I like that. Done.